src/odin/enhancedtrippath.cc

```diff
@@ -1110,6 +1110,19 @@ uint32_t EnhancedTripPath_Node::GetStraightestTraversableIntersectingEdgeTurnDeg
   return staightest_turn_degree;
 }
 
+bool EnhancedTripPath_Node::IsStraightestTraversableIntersectingEdgeReversed(
+    uint32_t from_heading,
+    const TripPath_TravelMode travel_mode) {
+  uint32_t straightest_traversable_xedge_turn_degree =
+      GetStraightestTraversableIntersectingEdgeTurnDegree(from_heading, travel_mode);
+  // Determine if the straightest intersecting edge is in the reversed direction
+  if ((straightest_traversable_xedge_turn_degree > 124) &&
```

Comment:
magic numbers...perhaps a constexpr

Reply:
agreed - I created an issue to fix all turn degree magic numbers https://github.com/valhalla/valhalla/issues/1739